.github/workflows/sycl-linux-run-tests.yml

```
@@ -286,10 +287,27 @@ jobs:
         -DDPCPP_INSTALL_DIR="$$(dirname (which clang++))/.." \
         $CMAKE_EXTRA_ARGS
         ninja -C build-cts
+
+    - name: [CTS] List devices
```

nit:
```suggestion
    - name: SYCL CTS List devices
```

Sorry, I didn't expand the context before.